pulp_rpm/app/models/advisory.py

```diff
@@ -251,7 +251,12 @@ class UpdateCollection(BaseModel):
     shortname = models.TextField(null=True)
     module = JSONField(null=True)
 
-    update_record = models.ManyToManyField(UpdateRecord, related_name="collections")
+    update_record = models.ForeignKey(UpdateRecord, related_name="collections",
+                                      on_delete=models.deletion.CASCADE)
+
+    class Meta:
+        constraints = [models.UniqueConstraint(fields=['name', 'update_record'],
```

Comment:
Django docs suggest to use UniqueConstraint instead of unique_together : https://docs.djangoproject.com/en/2.2/ref/models/options/#unique-together

Reply:
Great find!
I think we use `unique_together` everywhere and it's also used in identifying natural keys.
What do you think about staying consistent (using unique_together) but also filing a task to convert to `UniqueConstraint` in pulpcore and in plugins?

Reply:
Good point, consistency is a better choice - I'll change.

  I checked more-recent django doc, and 3.0 also says "may be deprecated" - so unique_together isn't going away any time soon. I don't think it's worth opening an issue/task at this point, we're not going to make this kind of change "just because", so it would just sit at NEW until it got auto-closed. Def something to think about for Pulp4 (or 5?)